content/sensu-go/5.14/api/apikey.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "APIKey API"
+description: "The APIKey API provides HTTP access to API key data. This is a reference for the APIKey API in Sensu Go, including examples for returning lists of API keys, creating API keys, and more. Read on for the full reference."
+version: "5.14"
+product: "Sensu Go"
+menu:
+  sensu-go-5.14:
+    parent: api
+---
+
+- [The `/apikey` API endpoint](#the-apikey-api-endpoint)
+	- [`/apikey` (GET)](#apikey-get)
+	- [`/apikey` (POST)](#apikey-post)
+- [The `/apikey/:apikey` API endpoint](#the-apikeyapikey-api-endpoint)
+	- [`/apikey/:apikey` (GET)](#apikeyapikey-get)
+  - [`/apikey/:apikey` (PATCH)](#apikeyapikey-patch)
+  - [`/apikey/:apikey` (DELETE)](#apikeyapikey-delete)
+
+
+## The `/apikey` API endpoint
+
+### `/apikey` (GET)
+
+The `/apikey` GET endpoint retrieves all API keys.
+
+#### EXAMPLE {#apikey-get-example}
+
+The following example demonstrates a request to the `/apikey` API, resulting in a JSON array containing all API keys.
```

Comment:
Need to confirm the response is a JSON array.

Reply:
Yes, it's an array. One might look like this:
```
[
    {
        "metadata": {
            "name": "my-api-key"
        },
        "key": "79190ae3-ca48-4e08-80e3-97e8c89af32e",
        "username": "user1",
        "groups": [
            "ops"
        ],
        "created_at": 1569874845
    },
    {
        "metadata": {
            "name": "my-other-api-key"
        },
        "key": "99990ae3-ca48-4e08-80e3-97e8c89af32e",
        "username": "user2",
        "groups": [
            "dev"
        ],
        "created_at": 1569874845
    }
]
```